server/sonar-server/src/main/java/org/sonar/server/activity/ActivityService.java

```diff
@@ -45,8 +46,15 @@ public void save(Activity activity) {
       .setAction(activity.getAction())
       .setMessage(activity.getMessage())
       .setData(KeyValueFormat.format(activity.getData()))
+      .setProfileKey((String) activity.getData().get("profileKey"))
```

Comment:
I thing this property should really be removed => having the same information twice is very misleading, and if for an unknown reason there are not the same, which one is right ?
There'll be a lot more impact if you leave it like this that if you remove it.